python/py-pytorch/Portfile

```diff
@@ -172,6 +177,14 @@ if {${name} ne ${subport}} {
                     USE_ZMQ=ON \
                     USE_ZSTD=OFF
 
+    # https://github.com/macports/macports-ports/pull/18009#issuecomment-1490276882
+    set fbgemm_minimum_supported_major_version 22
+    if {${os.platform} eq {darwin}
+        && ${os.major} < ${fbgemm_minimum_supported_major_version}} {
+        build.env-append \
+                    USE_FBGEMM=OFF
```

Comment:
Shouldn't we check if the removal of `-Werror` fixes the fbgemm configure check, before resorting to disabling entirely?

Reply:
`AVX512` support is required for `FBGEMM`, and @jmroot's [comment](https://github.com/macports/macports-ports/pull/18009#issuecomment-1490316527) above suggests that such support only exists on MacPorts's Ventura machines.

https://github.com/pytorch/FBGEMM/blob/177ba08dbc3bae56d11e4c89bdf19b04974374dc/CMakeLists.txt#L91

Reply:
The check that is failing is just testing whether compiling something with `-mavx512f` succeeds. My hypothesis is that there may be a warning generated in the test compile which is being turned into an error and making the test fail.

Reply:
Okay, thanks for the explanation. I'll let this CI complete then try a version with this part removed.

Reply:
> My hypothesis is that there may be a warning generated in the test compile which is being turned into an error and making the test fail.

Also, if your hypothesis is correct, wouldn't that imply a bug in upstream's `cmake` code?

Reply:
Yes, it would.